docs/community/3.9-announcement.md

```diff
@@ -179,13 +179,13 @@ There are a large number of minor fixes and improvements in this release. See th
 
 ## What's next
 
-We're planning to iteratively working towards OpenAPI becoming the standard schema
+We're planning to iteratively work towards OpenAPI becoming the standard schema
 representation. This will mean that the `coreapi` dependency will gradually become
 removed, and we'll instead generate the schema directly, rather than building
 a CoreAPI `Document` object.
 
 OpenAPI has clearly become the standard for specifying Web APIs, so there's not
-much value any more in our schema-agnostic document model. Making this change
+much value anymore in our schema-agnostic document model. Making this change
```

Comment:
Not sure here.

Reply:
I think this is technically correct, but reads awkwardly. Maybe due to the contraction on "there's"? How about
> so there isn't much value any longer ..